### DISCUSSION TITLE
about map

### DISCUSSION THREAD

**Discussion by zcswdt** [open-mmlab/mmdetection#9190] [Q&A] [answered]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmdetection/issues) and [Discussions](https://github.com/open-mmlab/mmdetection/discussions) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (master)](https://github.com/open-mmlab/mmdetection) or [latest version (3.x)](https://github.com/open-mmlab/mmdetection/tree/dev-3.x).

### 💬 Describe the reimplementation questions

Hello, I found that the map value has been very low during the training of target detection of a category, and it seems that the recall rate is normal. What is the reason for this?

![image](https://user-images.githubusercontent.com/43515926/198814798-5f233b22-ea07-4d1d-b7a1-8ca71e28e101.png)

### Environment

Pyhon3.7

**Comment by ZwwWayne** [comment, marked as answer]
This is because the model predicts too many objects. You may consider increase the score_thr in the test_cfg.

**Reply by zcswdt** [reply]
test_cfg In which file is this modified? Thank you
